autovalet_husky/src/autovalet_husky/find_parking_spot.py

```diff
@@ -160,7 +165,7 @@ def main():
     # tag_topic is the name of the topic in which the pose of the marker is being published
     # goal_pose_frame_id is the name of the frame in which you want the goal to be generated
     # aruco_frame_name is the NAME OF THE FRAME PUBLISHED AS THE ARUCO TAG
-    tag_topic          = '/aruco_single/pose'
+    tag_topic          = '/ARUCO/pose'
```

Comment:
not super strong, just me being anal. i renamed every node so that it looks like this when we launch

```
process[SLAM-2]: started with pid [1509]
process[NAVIGATION-3]: started with pid [1518]
process[ARUCO-4]: started with pid [1520]
process[nodelet_manager-5]: started with pid [1521]
process[nodelet1-6]: started with pid [1524]
process[STATE_MACHINE-7]: started with pid [1530]
```

more readable imo